pkg/kritis/metadata/containeranalysis/gen/doc.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// AUTO-GENERATED CODE. DO NOT EDIT.
+
+// Package containeranalysis is an auto-generated package for the
+// Container Analysis API.
+//
+//   NOTE: This package is in alpha. It is not stable, and is likely to change.
```

Comment:
Where did this get generated from?

Reply:
same place. https://storage.cloud.google.com/container-analysis-v1alpha1/containeranalysis-go.tar.gz
Will add in comment here too.

Reply:
Ah if we're taking generated code like this it needs to be in vendor. There's probably a way to grab this with go get/dep instead of doing it this way.

Reply:
i tried using dep but i keep getting an error
```
tejaldesai@tejaldesai~/go/src/github.com/grafeas/kritis: (add_container_ana_api)$ dep ensure
The following errors occurred while deducing packages:
  * "cloud.google.com/go/devtools/containeranalysis/apiv1alpha1": unable to deduce repository and source type for "cloud.google.com/go/devtools/containeranalysis/apiv1alpha1": unable to read metadata: go-import metadata not found

validateParams: could not deduce external imports' project roots
````

Since this code is not present here https://github.com/GoogleCloudPlatform/google-cloud-go or any other place. The client libraries are linked here https://cloud.google.com/container-registry/docs/vulnerability-scanning. 

I am happy to work around this with you. If i manually place this in vendor, things work but then dep ensure will error out next time.